lib/GenomeFileUtil/core/FastaGFFToGenome.py

```diff
@@ -708,9 +755,11 @@ def _aliases(feat):
 
         self.feature_dict[out_feat['id']] = out_feat
 
-    def _process_cdss(self):
+    def _process_cdss(self, prot_fasta_path):
         """Because CDSs can have multiple fragments, it's necessary to go
         back over them to calculate a final protein sequence"""
+        if self.is_metagenome:
```

Comment:
I would do `def _process_cdss(self, prot_fasta_path, is_metagenome)`. just in case some other sub func modify `self.is_metagenome` making it a debugging nightmare.

Reply:
Do you think changing `is_metagenome` from a class variable to one that is passed is worth it? I don't anticipate it being changed at all after it is initially declared.

Reply:
nahhhh. leaving it as class variable is fine. we will just watch out if any method changes it.